client/app/lib/__test__/redux-form.test.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { createStore, combineReducers } from 'redux';
+import { mount } from 'enzyme';
+import { reduxForm, reducer as formReducer, submit } from 'redux-form';
+import { SubmissionError } from 'lib/redux-form';
+
+describe('SubmissionError', () => {
+  describe('when no ActiveModel base errors are present', () => {
+    const errors = { title: 'is absent' };
+
+    it('makes no change', () => {
+      expect(new SubmissionError(errors).errors).toEqual(errors);
+    });
+  });
+
+  describe('when ActiveModel base errors are present', () => {
+    const baseError = 'Whole form wrong!';
+
+    describe('when _error is absent', () => {
+      const errors = { title: 'is absent', base: baseError };
+
+      it('copies ActiveModel base errors to it', () => {
+        expect(new SubmissionError(errors).errors).toEqual({ ...errors, _error: baseError });
+      });
+
+      it('is caught and displayed by redux-form', () => {
+        const DummyForm = reduxForm({ form: 'dummy' })(({ error, handleSubmit }) => (
+          <form onSubmit={handleSubmit}>
+            { error }
+          </form>
+        ));
+        const dummyStore = createStore(combineReducers({ form: formReducer }), {});
+        const wrapper = mount(
+          <DummyForm onSubmit={() => { throw new SubmissionError(errors); }} />,
+          buildContextOptions(dummyStore)
+        );
+
+        dummyStore.dispatch(submit('dummy'));
+        wrapper.update();
+        expect(wrapper.text()).toEqual(baseError);
+      });
+    });
+
+    describe('when _error is present', () => {
+      const errors = { title: 'is absent', base: baseError, _error: 'Existing top-level error.' };
+
+      it('does not overwrite or modify it', () => {
```

Comment:
Just a quick check, do we expect to override, or combine the errors?

Reply:
Actually, for the [normal use case](https://redux-form.com/7.3.0/docs/api/submissionerror.md/), no per-existing `_error` should be present. Decided to remove the unnecessary logic.